src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
-
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure figure1 = figureSupplier.getRandomFigure();
+        Figure figure2 = figureSupplier.getRandomFigure();
+        Figure figure3 = figureSupplier.getRandomFigure();
+        Figure figure4 = figureSupplier.getDefaultFigure();
+        Figure figure5 = new Circle("Black", 15);
+        Figure[] figures = new Figure[]{figure1, figure2, figure3, figure4, figure5};
```

Comment:
I think you can make it in for loop. [Hint](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#you-should-create-several-random-figures-so-you-will-use-a-loop-please-dont-create-a-new-figuresupplier-inside-the-loop).

Reply:
Also follow the task. The half of the array should be generated with `getDefaultFigure()`.